frontend/src/app/login/page.tsx

```
@@ -0,0 +1,203 @@
+
+          <div className="relative z-30 w-full max-w-[380px] -mt-12">
+            <div className="glass-card w-full rounded-2xl bg-white/80 p-7 shadow-[0_8px_32px_0_rgba(31,38,135,0.07)] backdrop-blur-xl border border-white/50">
+              <div className="mb-6 text-left">
+                <h3 className="mb-1.5 text-2xl font-bold text-text-main">Welcome!</h3>
+                <p className="text-text-sub text-xs">SearchWeb에 오신 것을 환영합니다.</p>
+              </div>
+
+              {/* Google Sign In - Purple Gradient Background */}
+              <button className="group relative mb-6 flex h-11 w-full items-center justify-center gap-3 rounded-lg border-t border-white/20 bg-[linear-gradient(135deg,#6d28d9,#8b5cf6)] text-white shadow-md transition-all hover:brightness-110 hover:shadow-primary/25">
+                <div className="flex h-6 w-6 items-center justify-center rounded-full bg-white p-1">
+                  <svg className="h-full w-full" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)